docs/troubleshooting.md

```diff
@@ -12,4 +12,51 @@ Troubleshooting Guide
 
 ## Management
 
-TODO
+### Heketi refuses to start and the log lines contain the string:
+
+    "Heketi was terminated while performing one or more operations."
+
+In Heketi 6.0 support for pending operations were added to the system.
+These pending operations are logged in the db before Heketi
+configures gluster and then are removed when the action is
+complete. However, if the Heketi process is terminated while it is
+performing these actions the pending operations will remain
+in the db. Because these stale pending operations exist in the db
+Heketi (by default) refuses to start, in order to avoid many of
+these stale operation entries from piling up and giving the
+administrator an opportunity to clean up any half-completed
+items on the gluster side.
+
+In order to start Heketi again one can:
+* Set the environment variable `HEKETI_IGNORE_STALE_OPERATIONS=1`
+  (The value may be any non-empty string).
```

Comment:
with to PR #1079, this turns into a Boolean variable and can take True and False as values.